src/corespotlight.cs

```diff
@@ -985,6 +985,19 @@ interface CSSearchableItemAttributeSet : NSCopying, NSSecureCoding {
 		[NullAllowed, Export ("supportsNavigation", ArgumentSemantic.Strong)]
 		NSNumber SupportsNavigation { get; set; }
 
+		[NoTV, NoMac, iOS (15,0), MacCatalyst (15,0)]
+		[Field ("CSActionIdentifier")]
+		NSString ActionIdentifier { get; }
+
+		[NoTV, NoMac, iOS (15,0), MacCatalyst (15,0)]
+		[Export ("actionIdentifiers", ArgumentSemantic.Copy)]
+		string[] ActionIdentifiers { get; set; }
```

Comment:
you expose `ActionIdentifier` as a `NSString`
but `ActionIdentifiers` as `string[]`
if it's to be used with constant then it's better to keep them all as `NSString`, it avoid conversions and remove the risk that the constants are compared using pointers

Reply:
ActionIdentifier is a Field that is used for the plist as per the docs:
```
Info dictionary of the specified NSUserActivity includes the corresponding Info.plist entry using the key CSActionIdentifier.
```
I think is safe to leave it as it is, maybe if anything move the plist to a string, but all other keys are NSStrings.